psycop/common/feature_generation/loaders/raw/load_ids.py

```diff
@@ -34,3 +39,46 @@ def load_ids(split: SplitName, n_rows: int | None = None) -> pd.DataFrame:
     df = sql_load(sql, database="USR_PS_FORSK", n_rows=n_rows)
 
     return df.reset_index(drop=True)
+
+
+def load_region_ids(split: SplitName, n_rows: int | None = None) -> pd.DataFrame:
+    """Loads ids for a given split using the region-based split.
+
+    Args:
+        split: Which split to load IDs from. Takes either "train", "test" or "val".
+        n_rows: Number of rows to return. Defaults to None.
+
+    Returns:
+        pd.DataFrame: Only dw_ek_borger column with ids
+    """
+    split_df = (
+        get_regional_split_df()
+        .filter(pl.col("split") == split.value)
+        .select("dw_ek_borger")
+        .collect()
+    )
+    if n_rows is not None:
+        split_df = split_df.head(n_rows)
+    return split_df.to_pandas()
+
+
+def load_ids(
+    split_type: Literal["original", "geographical"],
+    split_name: SplitName,
+    n_rows: int | None = None,
+) -> pd.DataFrame:
+    """Loads ids for a given split.
+
+    Args:
+        split_type: Which split to load IDs from. Takes either "original" or "geographical".
+        split_name: Which split to load IDs from. Takes either "train", "test" or "val".
+        n_rows: Number of rows to return. Defaults to None.
+
+    Returns:
+        pd.DataFrame: Only dw_ek_borger column with ids
+    """
+    if split_type == "original":
+        return load_original_ids(split=split_name, n_rows=n_rows)
+    if split_type == "geographical":
+        return load_region_ids(split=split_name, n_rows=n_rows)
+    raise ValueError(f"split_type {split_type} is not allowed")
```

Comment:
What's the advantage to having two separate functions?

Reply:
If we are keeping this, it looks like an excellent case for a match statement 👍 

(Then we can ignore the value-error)